server/historian/packages/historian-base/package.json

```diff
@@ -17,17 +17,19 @@
     "lint": "npm run eslint",
     "lint:fix": "npm run eslint:fix",
     "start": "node dist/www.js",
-    "test": "nyc --all --temp-directory nyc/.nyc_output --cache-dir nyc/.cache --report-dir nyc/report -x dist/test/**/* mocha dist/test",
```

Comment:
This should probably be moved to a "test:coverage" command rather than removed altogether. Similar to routerlicious packages (see [@fluidframework/server-services test:coverage](https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/services/package.json#L23))